sdk/core/core-http/review/core-http.api.md

```diff
@@ -519,6 +540,11 @@ export interface ParameterValue {
     value: any;
 }
 
+// Warning: (ae-forgotten-export) The symbol "ParsedWWWAuthenticate" needs to be exported by the entry point coreHttp.d.ts
+//
+// @public
+export function parseWWWAuthenticate(wwwAuthenticate: string): ParsedWWWAuthenticate;
```

Comment:
does this function need to be public?

Reply:
For now, until we move how we're testing. In chatting with Daniel, it'll take reworking how we're doing the test, so I'm planning splitting up this PR over a few PRs so I don't block it on that rework.

Reply:
Disregard. I thought about splitting it, but I'm seeing that this will move a lot of things public we don't want public. I'm going to ahead and see if I can move the tests that use this as well and avoid exposing what the tests need.